KTLOSNaxxLoot.json

```diff
@@ -47,13 +47,18 @@
 		}
 	},
 	"Boss Shared":{
+		"Dreadnaught 4 pc":{
+			"prio":["(1)", "(2)"],
+			"gp":"10",
+			"wowID":"22726"
+		},
 		"Splinter of Atiesh":{
 			"prio":["Moxie(2)","LC","EP/GP"],
 			"gp":"10",
 			"wowID":"22726"
 		},
 		"Desecrated Bracers":{
-			"prio":["Snydicate","Tanks","LC","EP/GP"],
+			"prio":["Snydicate(1)","Frau(2)","Tanks","BenDriller(2)","Rogues","LC","EP/GP"],
```

Comment:
This wording still seems strange compared to the other pieces.
I would think Snydi, Tug, Aisli would want this as part of their 4 piece, and Frau, Wizzo, Beeny for r2.
But in conjunction with the Dreadnaught 4pc on L50, what the actual order seems unclear.
What I would recommend is either:
1) Move the tier changes to another pull request
or
2) Leave all tier pieces as "Specific tanks -> Off Tanks (4 pc dreadnaught) -> Rogue priority"